streamz.js

```diff
@@ -152,18 +152,30 @@ Streamz.prototype._fn = function(d) {
   this.push(d);
 };
 
-Streamz.prototype._finalize = noop;
+Streamz.prototype._finalize = function() {
+  // In node 14 we need to look at `_writableState.buffered.length` - in older versions we look at `_writableState.length`
+  if (this.endedCb && !this._concurrent && (this._writableState.buffered && !this._writableState.buffered.length || !this._writableState.length)) {
```

Comment:
This used to only do this if incoming pipes was < 1.... but now it always does it?

Reply:
`endedCb` is only set if incomingPipes < 1  https://github.com/ZJONSSON/streamz/pull/25/commits/570d6f1e8f288e461b99016505cbe266dadb7376#diff-4823b7de66d03ac4621b2d4cd22ede1e1fd8df119a3fc293dc0e2f0421b024e0R167-R169

Reply:
oh interesting.  i didn't catch that.  so now it checks for incoming pipes < 1 after the last transform is finished, which is presumably better.